charter/index.html

```diff
@@ -28,269 +28,278 @@
     </style>
   </head>
   <body>
-    <p class="remove">{TBD: This is a draft template that Community or Business Groups MAY choose
-    to use for their Group Charter. You may edit your charter to remove or change
-    the text provided here. Community Groups SHOULD have charters as a way to build
-    shared understanding within a group about activities, and to attract new 
-    participants who appreciate a clear description of the group's scope and 
-    deliverables.}</p>
-    
-    <p class="remove">{TBD: The charter sections below include notes (marked with “{TBD”) 
-    on what to include and recommended text. Please remember to remove 
-    the notes in your specific charter.}</p> 
-  
+    <p class="remove">
+      {TBD: This is a draft template that Community or Business Groups MAY
+      choose to use for their Group Charter. You may edit your charter to
+      remove or change the text provided here. Community Groups SHOULD have
+      charters as a way to build shared understanding within a group about
+      activities, and to attract new participants who appreciate a clear
+      description of the group's scope and deliverables.}
+    </p>
+    <p class="remove">
+      {TBD: The charter sections below include notes (marked with “{TBD”) on
+      what to include and recommended text. Please remember to remove the notes
+      in your specific charter.}
+    </p>
     <h1>
       [DRAFT] WebVR Community Group Charter
     </h1>
-
-    <p><span class="remove">{TBD: remove next sentence before submitting for approval} </span>
-    This Charter is work in progress. To submit feedback, please use 
-    <span class="remove">{TBD:GitHub repository URL}</span> Issues where Charter is being developed.</p>
-    
+    <p>
+      <span class="remove">{TBD: remove next sentence before submitting for
+      approval}</span> This Charter is work in progress. To submit feedback,
+      please use <span class="remove">{TBD:GitHub repository URL}</span> Issues
```

Comment:
that should be https://w3c.github.io/webvr/charter/ (or https://github.com/w3c/charter/issues), right?